### PR TITLE
st patch: alpha

### PR DESCRIPTION
**Pull request by neeasade**
correction toggleable by xresources

I still get the weird border issue on this (ie, no border as shown [here](https://github.com/gnotclub/xst/issues/82#issuecomment-653828497)) but have yet to test in another setup

### CODE REVIEW

**Comment by actionless** [comment]
thanks a lot!

File: xst.c [hunk 1]
```diff
@@ -88,6 +88,7 @@ xrdb_load(void)
 		XRESOURCE_LOAD_INTEGER("borderpx", borderpx);
 		XRESOURCE_LOAD_INTEGER("borderless", borderless);
 		XRESOURCE_LOAD_INTEGER("cursorshape", cursorshape);
+		XRESOURCE_LOAD_INTEGER("disableAlphaCorrection", disableAlphaCorrection);
```
Comment (actionless) [inline xst.c:91]: you forgot to add `alpha` here

Comment (neeasade) [reply xst.c:91]: I'm thinking we can leave it as `opacity` at the xresources level

File: xst.c [hunk 2]
```diff
@@ -89,6 +89,7 @@ xrdb_load(void)
 		XRESOURCE_LOAD_INTEGER("borderless", borderless);
 		XRESOURCE_LOAD_INTEGER("cursorshape", cursorshape);
 		XRESOURCE_LOAD_INTEGER("disableAlphaCorrection", disableAlphaCorrection);
+		XRESOURCE_LOAD_INTEGER("opacity", alpha);
```
Comment (actionless) [inline xst.c:92]: shouldn't it be float?

Comment (neeasade) [reply xst.c]: hell yeah lol thanks

**Pull request merged**